### PR TITLE
Add humble support to CI.

### PR DESCRIPTION
**Pull request by clalancette**
Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

This is the beginning of Humble support in the CI scripts.  Note that this also requires changes to the Windows cookbooks, which are in https://github.com/ros-infrastructure/ros2-cookbooks/pull/45

### CODE REVIEW

**Review by audrow — APPROVED** [review]

**Review by ivanpauno — APPROVED** [review]
LGTM!

File: .gitmodules
Comment (clalancette) [reply .gitmodules:23]: > I guess this will have to be updated again before merging, right?

Yes, exactly.  Given that I have two approvals here, I'm going to go ahead and merge this (without this bit), then deploy.

**Pull request merged**
